lib/linguist/generated.rb

```diff
@@ -280,6 +281,16 @@ def node_modules?
       !!name.match(/node_modules\//)
     end
 
+    # Internal: Is the blob part of the Go vendor/ tree,
+    # not meant for humans in pull requests.
+    #
+    # Returns true or false.
+    def go_vendor?
+      return false unless extname == '.go'
```

Comment:
I started doing this too until I realized that it's possible `.md` and other files might also be placed here. I don't know if they filter out the dependencies saved to just `.go` files.